core-services/egov-indexer/src/main/java/org/egov/infra/indexer/service/IndexerService.java

```diff
@@ -171,8 +179,14 @@ public void validateAndIndex(String finalJson, String url, Index index) throws E
 	 */
 	public void indexWithESId(Index index, String finalJson) throws Exception {
 		StringBuilder urlForNonBulk = new StringBuilder();
-		urlForNonBulk.append(esHostUrl).append(index.getName()).append("/").append(index.getType()).append("/")
-				.append("_index");
+		//urlForNonBulk.append(esHostUrl).append(index.getName()).append("/").append(index.getType()).append("/").append("_index");
+		if(this.isLegacyVersionES) {
```

Comment:
@debasishchakraborty-egovt reverse this IF logic

Reply:
Check now!!

Reply:
@shivam-eGov